core/src/main/java/org/stellar/anchor/dto/sep38/InfoResponse.java

```diff
@@ -45,5 +45,37 @@ public InfoResponse(List<AssetInfo> assetInfoList) {
     private List<AssetInfo.Sep38Operation.DeliveryMethod> buyDeliveryMethods;
 
     private transient List<String> exchangeableAssetNames;
+
+    public boolean hasSellDeliveryMethod(String deliveryMethod) {
+      return hasDeliveryMethod(sellDeliveryMethods, deliveryMethod);
+    }
+
+    public boolean hasBuyDeliveryMethod(String deliveryMethod) {
+      return hasDeliveryMethod(buyDeliveryMethods, deliveryMethod);
+    }
+
+    private boolean hasDeliveryMethod(
+        List<AssetInfo.Sep38Operation.DeliveryMethod> deliveryMethods, String method) {
+      boolean noneIsAvailable = deliveryMethods == null || deliveryMethods.size() == 0;
+      boolean noneIsProvided = method == null || method.equals("");
+      if (noneIsAvailable && noneIsProvided) {
+        return true;
+      }
+
+      if (noneIsAvailable) {
+        return false;
+      }
+
+      if (noneIsProvided) {
+        return true;
+      }
```

Comment:
💡 Maybe it is better to use `supportsDeliveryMethod()` here, since we're returning `true` when `method` is null/empty.

Reply:
https://github.com/stellar/java-stellar-anchor-sdk/pull/113/commits/5cc3f2c1fcc743962adfb47f05033fe1445519fd